util/src/main/java/io/zeebe/util/health/FailureListener.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.util.health;
+
+public interface FailureListener {
```

Comment:
Considering this is a public interface that might be implemented by others, it might make sense to document the intent and its contract(s).